types/rpc-handler.ts

```diff
@@ -12,7 +12,7 @@ export class RPCHandler implements HandlerInterface {
   private _networkName: string;
   private _env: string = "node";
 
-  private _rpcTimeout: number = 500; // ms
+  private _rpcTimeout: number = 999999; // ms
```

Comment:
```suggestion
  private _rpcTimeout: number = Number.MAX_VALUE; // ms
```

Why not just use this

Reply:
My dumbass thinking well 999999 is a better timeout than `Number.MAX_VALUE` because it's lesser, but it doesn't look as neat does it